atecc608a/main.c

```diff
@@ -26,7 +26,7 @@
 #include "atecc608a_utils.h"
 #include "atca_helpers.h"
 #include "atecc508a_config_dev.h"
-
+#include "psa_crypto_storage.h"
```

Comment:
This is a private interface. Can we use `mbed_psa_reboot_and_request_new_security_state(PSA_LIFECYCLE_ASSEMBLY_AND_TEST);` instead? This comes from "psa/lifecycle.h"

Reply:
I experimented with the function you suggested, and It works. Upon Andrzej's request I manually ran the test few times in a row, and they still pass.

Temporary removal of the reset code caused the tests to fail, so we made sure this step is necessary and the solution is valid.